Library/Homebrew/os/mac/pkgconfig/12/libcurl.pc

```diff
@@ -0,0 +1,40 @@
+#***************************************************************************
+#                                  _   _ ____  _
+#  Project                     ___| | | |  _ \| |
+#                             / __| | | | |_) | |
+#                            | (__| |_| |  _ <| |___
+#                             \___|\___/|_| \_\_____|
+#
+# Copyright (C) 2001 - 2018, Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# This software is licensed as described in the file COPYING, which
+# you should have received as part of this distribution. The terms
+# are also available at https://curl.haxx.se/docs/copyright.html.
+#
+# You may opt to use, copy, modify, merge, publish, distribute and/or sell
+# copies of the Software, and permit persons to whom the Software is
+# furnished to do so, under the terms of the COPYING file.
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+# KIND, either express or implied.
+#
+###########################################################################
+
+# This should most probably benefit from getting a "Requires:" field added
+# dynamically by configure.
+#
+homebrew_sdkroot=/Library/Developer/CommandLineTools/SDKs/MacOSX12.sdk
+prefix=${homebrew_sdkroot}/usr
+exec_prefix=/usr
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+supported_protocols="DICT FILE FTP FTPS GOPHER HTTP HTTPS IMAP IMAPS LDAP LDAPS POP3 POP3S RTSP SMB SMBS SMTP SMTPS TELNET TFTP"
+supported_features="AsynchDNS IPv6 Largefile GSS-API Kerberos SPNEGO MultiSSL NTLM NTLM_WB SSL libz HTTP2 UnixSockets HTTPS-proxy"
+
+Name: libcurl
+URL: https://curl.haxx.se/
+Description: Library to transfer files with ftp, http, etc.
+Version: 7.64.1
```

Comment:
```suggestion
supported_protocols="DICT FILE FTP FTPS GOPHER HTTP HTTPS IMAP IMAPS LDAP LDAPS MQTT POP3 POP3S RTSP SMB SMBS SMTP SMTPS TELNET TFTP"
supported_features="AsynchDNS GSS-API HTTPS-proxy IPv6 Kerberos Largefile MultiSSL NTLM NTLM_WB SPNEGO SSL UnixSockets alt-svc libz"

Name: libcurl
URL: https://curl.se/
Description: Library to transfer files with ftp, http, etc.
Version: 7.76.0
```

(Side note: a bit odd they removed HTTP2 support?)

Reply:
Almost all major browsers have [support][] now, even if it is still a draft according to IETF. It's behind a flag if you're running Big Sur, but it's been in the Technology Preview for a while and could change with Monterey.

(Not running it - just a guess)

A few libraries and servers are still working on support - notably libcurl - but most seem to be ready.

[support]: https://caniuse.com/http3

Reply:
Yeah I get that, but it was present as a "supported feature" in Curl on Big Sur but appears to have disappeared entirely when I checked the Monterey SDK.

(Comparing `curl-config --features` between 11.3 and 12.0 SDKs)

Another sign was `otool -L /usr/lib/libcurl.4.dylib` linking to `/usr/lib/libapple_nghttp2.dylib` but I don't have a 12.0 install to check.

Reply:
I remember curl does have HTTP2 feature in the new CLT on macOS 12. But now I don't have access to it anymore so I cannot check.